crates/bevy_render/src/render_resource/bind_group_macros.rs

```diff
@@ -0,0 +1,55 @@
+#[doc(hidden)]
+#[macro_export]
+macro_rules! bind_group_descriptor_acc {
+    (@d texture($e:expr)) => ($crate::render_resource::BindingResource::TextureView($e));
+    (@d sampler($e:expr)) => ($crate::render_resource::BindingResource::Sampler($e));
+    (@d buffer($e:expr)) => ($e);
+    (@d $_:ident($e:expr)) => (compile_error!("only 'texture', 'sampler', and 'buffer' bindings are supported"));
```

Comment:
this could be

```suggestion
    (@d $i:ident($e:expr)) => (compile_error!(concat!("only 'texture', 'sampler', and 'buffer' are supported, found '", stringify!($i), "'")));
```